rpc/grpc/plugin_impl_grpc.go

```diff
@@ -82,8 +82,7 @@ func (plugin *Plugin) Init() error {
 	if plugin.listenAndServe != nil {
 		plugin.netListener, err = plugin.listenAndServe(*plugin.grpcCfg, plugin.grpcServer)
 	} else {
-		plugin.Log.Info("Listening GRPC on tcp://", plugin.grpcCfg.Endpoint)
-		plugin.netListener, err = ListenAndServeGRPC(plugin.grpcCfg, plugin.grpcServer)
+		plugin.netListener, err = ListenAndServeGRPC(plugin.grpcCfg, plugin.grpcServer, plugin.Log)
```

Comment:
I don't understand how this relates to the unix domain socket.  This patch looks like two things (explicit adding of Logger to listen and serve and unix file socket got mushed together).

Reply:
Also, why in the default case do we want to force folks to manually pass round Loggers?